kinds/structured/folder/folder_kind.cue

```diff
@@ -0,0 +1,48 @@
+package kind
+
+name: "Folder"
+maturity: "merged"
+
+lineage: seqs: [
+	{
+		schemas: [
+			{//0.0
+				// OrgId is the ID of an organisation the folder belongs to.
+				orgId: int64 @grafanamaturity(ToMetadata="sys")
+				// Uid is the unique numeric identifier for the folder.
+				uid: string
+				// Title of the folder.
+				title: string
+				// TODO
+				url: string
```

Comment:
probably optional?

Reply:
`url`? Not sure, gonna take a look how it's used in the code.

Reply:
Are we trying to replicate/replace the existing tangled mess?

Reply:
We're following the [maximalist field selection rule](https://docs.google.com/document/d/1gPzSCQn-0Qg86PYpPcR-l_kTQ4Ba1wgnNLZFKPSe8yQ/edit#heading=h.jrv2hv50f2cu), per [the WIP guide](https://docs.google.com/document/d/1X_R-R5LIhGoqvZUec7tQyaPXduQW_qivWJjFao5-LJ4/edit#heading=h.am98mp3y9asd) procedure for arriving at `merged` maturity.

The [`@grafanamaturity` attributes](https://docs.google.com/document/d/1gPzSCQn-0Qg86PYpPcR-l_kTQ4Ba1wgnNLZFKPSe8yQ/edit#heading=h.iueyrgykhe1k) indicate the plan for how most of the fields are likely to go away. The remainder can be discussed in a follow-up PR.

Comparing to what you linked, i suppose we should add a `@grafanamaturity(MaybeRemove)` to `url`, here? Or if it's supposed to be a computed value, it should be `@grafanamaturity(ToMetadata="kind")` according to what we outlined in #59399